File: Typescript/test/alternatingSum.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/alternatingSum';

test('Example 1', () => {
	let input = [1, 3, 5, 7];
	let expected = -4;
	expect(solution.alternatingSum(input)).toEqual(expected);
});

test('Example 2', () => {
	let input = [100];
	let expected = 100;
	expect(solution.alternatingSum(input)).toEqual(expected);
});

File: Dart/test/alternatingSum_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/alternatingSum.dart';

void main() {
  test("test example 1", () {
    var input = [1, 3, 5, 7];
    var expected = -4;

    final sln = Solution();
    var output = sln.alternatingSum(input);
    expect(output, expected);
  });
  test("test example 2", () {
    var input = [100];
    var expected = 100;

    final sln = Solution();
    var output = sln.alternatingSum(input);
    expect(output, expected);
  });
}
